src/MoBi.Presentation/Presenter/CreateObjectPathsFromReferencesPresenter.cs

```diff
@@ -14,6 +14,8 @@ public interface ICreateObjectPathsFromReferencesPresenter : IPresenter<ICreateO
       IReadOnlyList<ObjectPath> GetAllSelections();
       void AddSelection();
       Size? ModalSize { get; }
+      void SetLayoutItemLocalisationVisible(bool visible);
+      void SetRelativePathRadioButtonEnabled(bool enabled);
```

Comment:
I am wondering about naming properties after the ux element that they represent. I think the fact that it’s a radio button is irrelevant. The presenter should worry about the function being shown or hidden, not how the view is displaying it. In fact, I have no idea what this code is doing without looking at the UI and understanding exactly how it’s organized 

Also agreed with Juri. Why do we need two properties ?